src/views/Landing.vue

```diff
@@ -0,0 +1,1172 @@
+                  invited – to take a peek under the hood and use the source code, fork it, or run
+                  and operate it locally. Superhero relies on its community to ensure utmost
+                  security while developing cool new features and moving the ecosystem forward. Join
+                  the global collaborative platform today and be a part of the change!
+                </p>
+                <a
+                  class="sh-btn-blue" href="https://github.com/aeternity/superhero-ui/"
+                  target="_blank"
+                ><img alt="" src="../assets/landing/github.svg"> Contribute on Github</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="superhero-cta button-left-section">
+        <img alt="" src="../assets/landing/superhero-cta.svg">
+        <div class="sh-join">
+          <RouterLink
+            :to="{ name: 'feed' }"
+            class="sh-btn-blue"
+          >
+            {{ $t('views.Landing.section1.linkFeed') }}
+          </RouterLink>
+        </div>
+      </section>
+      <section class="superhero-footer">
+        <div class="sh-container bottom-footer top">
+          <div class="sh-row">
+            <div class="sh-col-6 left-foot">
+              <p>Superhero is the world’s first decentralized, open-source social media
+                platform.</p>
+            </div>
+            <div class="sh-col-6 right-subscribe">
+              <form class="subscribe-form">
+                <div class="form-group">
+                  <input class="form-control" name="" placeholder="Get started" type="text">
+                  <button class="subscribe-button">Subscribe now</button>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+        <div class="sh-container bottom-footer">
+          <div class="sh-row">
+            <div class="sh-col-4 left-sh-foot">
+              <a href=""><img alt="" src="../assets/landing/superhero-logo-black.svg"></a>
+            </div>
+            <div class="sh-col-4 center-sh-foot">
+              <p>© 2020 Superhero.com</p>
+            </div>
+            <div class="sh-col-4 right-sh-foot">
+              <a href="">Privacy</a>
+              <a href="">Terms</a>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+  </div>
+</template>
+<script>
+import VueAos from 'vue-aos';
+
+export default {
+  components: { VueAos },
+  metaInfo: {
+    title: 'Welcome',
+  },
+};
+</script>
+<style lang="scss">
+:root {
+  --animate-delay: 0.1s; /* Animation delay 10x faster */
+  --animate-duration: 0.5s;
+}
+
+.landing-page {
+  background: #f2f2f2;
+  color: #212529;
+  font-family: Oxygen, sans-serif;
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 0 auto;
+  max-width: 100vw !important;
+  text-align: left;
+}
+</style>
+<style lang="scss" scoped>
+a {
```

Comment:
```suggestion
.landing { // should point to the root node of the component
  a {
```
to improve performance according to https://vue-loader.vuejs.org/guide/scoped-css.html#also-keep-in-mind